### ISSUE TITLE
Versions are incorrectly sorted

### ISSUE THREAD

**Issue by toebeann** [citkane/typedoc-plugin-versions#11]
As seen [here](https://procedure-rpc.github.io/procedure.js):
![image](https://user-images.githubusercontent.com/45315526/188323873-dd7264ed-628d-4655-b8a7-5b53c39cf1c1.png)

**Comment by citkane** [comment]
All awsome stuff @toebeann , [9](https://github.com/citkane/typedoc-plugin-versions/pull/9) and [10](https://github.com/citkane/typedoc-plugin-versions/pull/10) both.

I have code reviewed 9 and left a very minor comment, after which I will merge into dev.

My other job has got a bit busy, so give me a while to review 10, but i am agreed in principle. I want to add your logic into the user documentation as well.

We can then release it all into 0.2.0.

Thanks!

Do you have any interest in co-maintaining?

**Comment by toebeann** [comment]
Re. reviewing #10, it will certainly be easier to review once #9 is merged (assuming that is agreeable). In the meantime I have added a PR on my own fork that allows you to easily view the diffs between #9 and #10, this PR is available here: https://github.com/toebeann/typedoc-plugin-versions/pull/1

Re. docs, in #10 I added some minor readme changes to signify the existence of automatic inferral, but you're right - it would be wise to elaborate on this further in the readme. I'll get on that this eve

Re. co-maintaining, fine by me though no promises as to how much of my time and energy going forward I can commit to it, but I'm always happy to help out on useful projects when i can 🙂
